src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,10 +4,15 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
+        logger.info("Method completeOrder with this userId = {}, was called", userId);
         // TODO: add log message about method completeOrder was called
```

Comment:
According to check-list you need to delete all commented lines.

Reply:
ok